src/libmongoc/examples/cmake/vcpkg/.gitignore

```diff
@@ -0,0 +1 @@
+_build/
```

Comment:
```suggestion
_build/

```
Missing EOF newline.

The root `.gitignore` file already contains `_build/`; does that not apply here?

Reply:
Yeah, this file can be removed.